### ISSUE TITLE
uint64_t value at Variable view treated as signed value

### ISSUE THREAD

**Issue by CodeExamplesTutorials** [eclipse-cdt/cdt#763]
**Describe the bug**
A uint64_t variable is shown as signed value in debug mode at the variable view.

**To Reproduce**
create a Variable of uint64_t and assign e.g. 0xFFFFFFFFFFFFFFFF.
Start debugging and open Variable View. Decimal Value shown as -1

**Expected behavior**
Expected the value 18446744073709551615 because variable is unsigned.

**Screenshots**
see screenshot

**Version Information (please complete the following information):**
 - Tested on Windows 11 and Ubuntu 22.04
 - Eclipse Version 2024-03 (4.31.0)
 - CDT Version: 11.5.0.202402230238 - Build id: 20240304-1627
![Eclipse_uint64_t_debug](https://github.com/eclipse-cdt/cdt/assets/166758642/d6bce10c-7264-4946-af37-94424e0bd1c7)

**Comment by jld01** [comment]
Hi @CodeExamplesTutorials, the presentation formats you observe in the _Variables_ view are those provided directly by GDB/MI. When CDT requests a value in decimal format, GDB/MI always provides a signed decimal response (equivalent to the GDB `print /d` command). GDB/MI does not offer an unsigned decimal format (equivalent to the GDB `print /u` command) at present.

Does this explanation resolve the issue for you?

**Comment by CodeExamplesTutorials** [comment]
Yes correct, this issue is related to GDB. I saw this on my Windows work PC for uint8_t / uint16_t and uint32_t displayed correct but not for uint64_t. I just made this simple example for uint64_t on my linux PC. Now I have extended the example on my Linux PC. On the Linux PC also uint8_t / uint16_t and uint32_t is displayed at -1. The only difference is the gdb version. The gdb on the Windows PC was installed by mingw is verry old.
